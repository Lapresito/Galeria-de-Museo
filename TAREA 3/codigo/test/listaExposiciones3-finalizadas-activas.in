crearListaExposiciones
crearExposicion 01/01/2002 01/05/2002 10 1
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
crearExposicion 01/02/2002 01/06/2002 10 2
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
crearExposicion 01/03/2002 01/07/2002 10 3
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
obtenerExposicionesFinalizadas 02/05/2002
imprimirListaExposiciones
obtenerExposicionesFinalizadas 02/06/2002
imprimirListaExposiciones
obtenerExposicionesFinalizadas 02/07/2002
imprimirListaExposiciones
liberarListaExposiciones
crearListaExposiciones
crearExposicion 01/01/2002 01/07/2002 10 1
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
crearExposicion 01/02/2002 01/07/2002 10 2
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
crearExposicion 02/03/2002 01/07/2002 10 3
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
imprimirListaExposiciones
obtenerExposicionesFinalizadas 02/08/2002
imprimirListaExposiciones
liberarListaExposiciones
crearListaExposiciones
crearExposicion 01/01/2002 01/07/2002 10 1
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
crearExposicion 01/02/2002 01/07/2002 10 2
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
crearExposicion 02/03/2002 01/07/2002 10 3
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
obtenerExposicionesActivas 15/01/2002
imprimirListaExposiciones
obtenerExposicionesActivas 15/02/2002
imprimirListaExposiciones
obtenerExposicionesActivas 15/03/2002
imprimirListaExposiciones
liberarListaExposiciones
crearListaExposiciones
crearExposicion 01/01/2002 01/07/2002 10 1
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
crearExposicion 01/02/2002 01/07/2002 10 2
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
crearExposicion 02/03/2002 01/07/2002 10 3
agregarAExposicion 10 0 1 2 3 4 5 6 7 8 9
agregarAListaExposiciones
obtenerExposicionesActivas 15/03/2002
imprimirListaExposiciones
liberarListaExposiciones
Fin