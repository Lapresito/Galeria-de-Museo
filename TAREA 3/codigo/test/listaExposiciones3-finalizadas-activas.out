Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Lista de exposiciones creada con exito.
2> Exposicion creada con exito.
3> Piezas agregadas a la exposicion de forma exitosa.
4> Se ha agregado la exposicion a la lista de forma exitosa.
5> Exposicion creada con exito.
6> Piezas agregadas a la exposicion de forma exitosa.
7> Se ha agregado la exposicion a la lista de forma exitosa.
8> Exposicion creada con exito.
9> Piezas agregadas a la exposicion de forma exitosa.
10> Se ha agregado la exposicion a la lista de forma exitosa.
11> Exposiciones finalizadas: 
Exposicion #1 del 1/1/2002 al 1/5/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
12> Exposicion #2 del 1/2/2002 al 1/6/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
Exposicion #3 del 1/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
13> Exposiciones finalizadas: 
Exposicion #2 del 1/2/2002 al 1/6/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
14> Exposicion #3 del 1/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
15> Exposiciones finalizadas: 
Exposicion #3 del 1/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
16> 17> Lista exposiciones liberada con exito.
18> Lista de exposiciones creada con exito.
19> Exposicion creada con exito.
20> Piezas agregadas a la exposicion de forma exitosa.
21> Se ha agregado la exposicion a la lista de forma exitosa.
22> Exposicion creada con exito.
23> Piezas agregadas a la exposicion de forma exitosa.
24> Se ha agregado la exposicion a la lista de forma exitosa.
25> Exposicion creada con exito.
26> Piezas agregadas a la exposicion de forma exitosa.
27> Se ha agregado la exposicion a la lista de forma exitosa.
28> Exposicion #1 del 1/1/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
Exposicion #2 del 1/2/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
Exposicion #3 del 2/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
29> Exposiciones finalizadas: 
Exposicion #1 del 1/1/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
Exposicion #2 del 1/2/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
Exposicion #3 del 2/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
30> 31> Lista exposiciones liberada con exito.
32> Lista de exposiciones creada con exito.
33> Exposicion creada con exito.
34> Piezas agregadas a la exposicion de forma exitosa.
35> Se ha agregado la exposicion a la lista de forma exitosa.
36> Exposicion creada con exito.
37> Piezas agregadas a la exposicion de forma exitosa.
38> Se ha agregado la exposicion a la lista de forma exitosa.
39> Exposicion creada con exito.
40> Piezas agregadas a la exposicion de forma exitosa.
41> Se ha agregado la exposicion a la lista de forma exitosa.
42> Exposiciones activas: 
Exposicion #1 del 1/1/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
43> Exposicion #2 del 1/2/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
Exposicion #3 del 2/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
44> Exposiciones activas: 
Exposicion #2 del 1/2/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
45> Exposicion #3 del 2/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
46> Exposiciones activas: 
Exposicion #3 del 2/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
47> 48> Lista exposiciones liberada con exito.
49> Lista de exposiciones creada con exito.
50> Exposicion creada con exito.
51> Piezas agregadas a la exposicion de forma exitosa.
52> Se ha agregado la exposicion a la lista de forma exitosa.
53> Exposicion creada con exito.
54> Piezas agregadas a la exposicion de forma exitosa.
55> Se ha agregado la exposicion a la lista de forma exitosa.
56> Exposicion creada con exito.
57> Piezas agregadas a la exposicion de forma exitosa.
58> Se ha agregado la exposicion a la lista de forma exitosa.
59> Exposiciones activas: 
Exposicion #1 del 1/1/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
Exposicion #2 del 1/2/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
Exposicion #3 del 2/3/2002 al 1/7/2002
Piezas: 0 1 2 3 4 5 6 7 8 9 
60> 61> Lista exposiciones liberada con exito.
62> Fin.
